src/modules/markets/selectors/market.js

```diff
@@ -356,6 +360,11 @@ export function assembleMarket(
             .filter(collection => collection.length !== 0)
             .flat() || [];
 
+        // add pending orders
+        if (pendingOrders && pendingOrders.length > 0) {
+          market.userOpenOrders = market.userOpenOrders.concat(pendingOrders);
```

Comment:
should pending orders go first so they show up in the lists first?

Reply:
yes, they should. making change

Reply:
I see your comment and raise you a commit.